src/Avalonia.Controls.TreeDataGrid/Automation/Peers/TreeDataGridCellAutomationPeer.cs

```diff
@@ -0,0 +1,23 @@
+using Avalonia.Automation.Peers;
+using Avalonia.Controls.Primitives;
+
+namespace Avalonia.Controls.Automation.Peers;
+
+public class TreeDataGridCellAutomationPeer : ControlAutomationPeer
```

Comment:
Shouldn't cell be a content control automation peer? Or just override GetNameCore 
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Automation/Peers/ContentControlAutomationPeer.cs#L16-L31

Reply:
https://github.com/AvaloniaUI/Avalonia.Controls.TreeDataGrid/pull/332/commits/b02607e6204212e9cb967f2ea310cb72edb71240